traits.yml

```diff
@@ -91,26 +77,34 @@ specifications:
             name: Raster
           - namespace: image
             name: OCIOColorManaged
-          - namespace: content
+          - package: openassetio-mediacreation
```

Comment:
Because these ones are in the media creation package, they need the package name as well, otherwise itll try to load them from this package.